diff --git a/1_create_package/nomad/put_nomad_binary_here b/1_create_package/nomad/put_nomad_binary_here
deleted file mode 100644
index e69de29..0000000
diff --git a/1_create_package/ui/config b/1_create_package/ui/config
index 2215f01..25c2678 100644
--- a/1_create_package/ui/config
+++ b/1_create_package/ui/config
@@ -8,7 +8,7 @@
             "icon": "nomad-{0}.png",
             "protocol": "http",
             "url": "/",
-            "port": "4646", 
+            "port": "{{ nidito.nomad.ports.http }}", 
             "grantPrivilege": "local"
         }
     }
diff --git a/2_create_project/INFO b/2_create_project/INFO
index c218db0..2c4a3dd 100644
--- a/2_create_project/INFO
+++ b/2_create_project/INFO
@@ -1,5 +1,5 @@
 package="nomad"
-version="0.12.4"
+version="{{ nidito.nomad.version }}"
 description="Easily Deploy Applications at Any Scale"
 support_url="https://github.com/hashicorp/nomad/"
 displayname="nomad"
diff --git a/2_create_project/scripts/nomad.sc b/2_create_project/scripts/nomad.sc
index 3fea758..e84bab4 100755
--- a/2_create_project/scripts/nomad.sc
+++ b/2_create_project/scripts/nomad.sc
@@ -2,5 +2,5 @@
 title="nomad"
 desc="nomad"
 port_forward="yes"
-dst.ports="4646/tcp 4647/tcp 4648/tcp"
+dst.ports="{{ nidito.nomad.ports.http }}/tcp {{ nidito.nomad.ports.rpc }}/tcp {{ nidito.nomad.ports.serf }}/tcp {{ nidito.nomad.ports.serf }}/udp"
 
