diff --git a/1_create_package/consul/put_consul_binary_here b/1_create_package/consul/put_consul_binary_here
deleted file mode 100644
index e69de29..0000000
diff --git a/1_create_package/ui/config b/1_create_package/ui/config
index 6d269fd..c3f4f3a 100644
--- a/1_create_package/ui/config
+++ b/1_create_package/ui/config
@@ -8,7 +8,7 @@
             "icon": "consul-{0}.png",
             "protocol": "http",
             "url": "/",
-            "port": "8500",
+            "port": "{{ nidito.consul.ports.http }}",
             "grantPrivilege": "local"
         }
     }
diff --git a/2_create_project/INFO b/2_create_project/INFO
index b1ae15d..8665eb6 100644
--- a/2_create_project/INFO
+++ b/2_create_project/INFO
@@ -1,5 +1,5 @@
 package="consul"
-version="1.8.4"
+version="{{ nidito.consul.version }}"
 description="Consul is a distributed, highly available, and data center aware solution to connect and configure applications across dynamic, distributed infrastructure."
 support_url="https://github.com/hashicorp/consul/"
 displayname="consul"
@@ -8,7 +8,7 @@ maintainer_url="https://consul.io"
 distributor="Sebastien Bariteau"
 distributor_url="https://github.com/numkem/consul-spk"
 arch="noarch"
-adminport=8500
+adminport={{ nidito.consul.ports.http }}
 adminurl="ui"
 dsmuidir="ui"
 checkport="yes"
diff --git a/2_create_project/scripts/consul.sc b/2_create_project/scripts/consul.sc
index b91e74a..8e2481f 100755
--- a/2_create_project/scripts/consul.sc
+++ b/2_create_project/scripts/consul.sc
@@ -2,5 +2,5 @@
 title="consul"
 desc="consul"
 port_forward="yes"
-dst.ports="8500/tcp 8501/tcp"
+dst.ports="{{ nidito.consul.ports.http }}/tcp {{ nidito.consul.ports.serf_lan }},{{ nidito.consul.ports.serf_wan }}/tcp,udp {{ nidito.consul.ports.dns }}/udp"

